asserts/crypto.go

```diff
@@ -0,0 +1,36 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2014-2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package asserts
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"time"
+
+	"golang.org/x/crypto/openpgp/packet"
+)
+
+func generatePrivateKey() (*packet.PrivateKey, error) {
+	priv, err := rsa.GenerateKey(rand.Reader, 2048)
+	if err != nil {
+		return nil, err
+	}
+	return packet.NewRSAPrivateKey(time.Now(), priv), nil
```

Comment:
The following is a bit of a nitpick and you are free to ignore. However the rest of the code is (more or less) following it (and Sergio was keen in his reviews that we follow), so it might help with the consistency (and after you get used to it also the readability).

Could you please add a "\n" before the return line?  I.e.
```
}

return ...
```
There is a pattern in the current code that stuff that does not strongly needs to be together is separated by a empty line with just \n. I will add some more \n suggestions below. But if you or @niemeyer do not like this just ignore please.